ui.af.apps/src/main/content/jcr_root/apps/core/fd/components/form/dropdown/v1/dropdown/clientlibs/site/css/dropdownview.css

```diff
@@ -22,6 +22,10 @@
 
 }
 
+.cmp-adaptiveform-dropdown__widget[multiple='multiple'] {
```

Comment:
I'd prefer it being part of the theme. we can fix this in the next release.

Reply:
lets remove the height from this css.

Reply:
yes

Reply:
This is the default height of multi-select like textarea. Users can also override it through the theme.

Reply:
removed